client/capital/index.tsx

```diff
@@ -218,6 +218,7 @@ const CapitalPage = (): JSX.Element => {
 				totalRows={ loans.length }
 				headers={ columns }
 				rows={ getRowsData( loans ) }
+				rowsPerPage={ loans.length }
```

Comment:
Note that I also checked that server returns _all_ loans, so pagination is not a concern for this view.